ui/common/src/socket.ts

```diff
@@ -41,47 +41,50 @@ interface Settings {
   params?: Partial<Params>;
   options?: Partial<Options>;
 }
-
-const origSend = WebSocket.prototype.send;
+// TODO - find out why are there three different types of settings
 
 const isOnline = () => !('onLine' in navigator) || navigator.onLine;
 
-// versioned events, acks, retries, resync
-export default class StrongSocket {
-  settings: Settings;
-  options: Options;
-  version: number | false;
-  ws: WebSocket | undefined;
-  pingSchedule: Timeout;
-  connectSchedule: Timeout;
-  ackable: Ackable = new Ackable((t, d, o) => this.send(t, d, o));
-  lastPingTime: number = performance.now();
-  pongCount = 0;
+export default class StrongSocket implements SocketI {
   averageLag = 0;
-  tryOtherUrl = false;
-  autoReconnect = true;
-  nbConnects = 0;
-  storage: LichessStorage = storage.make(
```

Comment:
fix broken `this.storage` assignment which never actually created 'surl-alt' because `socket-alternates` is not a valid data attribute, we have been using `socket-alts`